amenity-points.mss

```diff
@@ -1254,6 +1228,43 @@
     text-placement: interior;
   }
 
+  [feature = 'natural_peak'][zoom >= 11],
+  [feature = 'natural_volcano'][zoom >= 11],
+  [feature = 'natural_saddle'][zoom >= 13] {
+      [feature = 'natural_peak'] {
+        shield-file: url('symbols/peak.svg');
+      }
+      [feature = 'natural_volcano'] {
+        shield-file: url('symbols/volcano.svg');
+      }
+      [feature = 'natural_saddle'] {
+        shield-file: url('symbols/saddle.svg');
+      }
+      shield-name: '[name]';
+      shield-face-name: @book-fonts;
+      shield-fill: darken(@landform-color, 30%);
+      [feature = 'natural_volcano'] {
+        shield-fill: #d40000;
+      }
+      shield-size: 9;
+      shield-text-dy: 7;
+      shield-text-dx: 8;
+
+      [zoom >= 13] {
+        shield-size: 10;
+        shield-text-dy: 8;
+        shield-text-dx: 9;
+      }
+      shield-min-distance: 50;
```

Comment:
Wouldn't this also block highway shields near peaks, and peaks near highway shields?